Source/WebCore/html/CheckboxInputType.cpp

```diff
@@ -104,23 +104,25 @@ void CheckboxInputType::handleKeyupEvent(KeyboardEvent& event)
 
 void CheckboxInputType::handleMouseDownEvent(MouseEvent& event)
 {
-    ASSERT(element());
+    RefPtr element = this->element();
+    ASSERT(element);
 
-    if (!event.isTrusted() || !isSwitch() || element()->isDisabledFormControl() || !element()->renderer())
+    if (!event.isTrusted() || !isSwitch() || element->isDisabledFormControl() || !element->renderer())
         return;
     startSwitchPointerTracking(event.absoluteLocation());
 }
 
 void CheckboxInputType::handleMouseMoveEvent(MouseEvent& event)
 {
-    ASSERT(element());
+    RefPtr element = this->element();
```

Comment:
This early variable can be moved *after* the `!isSwitchPointerTracking()` early return.

Reply:
Sure, done.